src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -149,6 +149,53 @@ public ResponseEntity<Void> modifyIdentifierContingencyList(
         }
     }
 
+    @PostMapping(value = "/filters-contingency-lists", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create an filter base contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filter based contingency list has been created successfully")})
+    public ResponseEntity<PersistentContingencyList> createFilterBasedContingencyList(@RequestParam(required = false, value = "id") UUID id,
+                                                                                     @RequestBody FilterBasedContingencyList filterBasedContingencyList) {
+        return ResponseEntity.ok()
+            .contentType(MediaType.APPLICATION_JSON)
+            .body(service.createFilterBasedContingencyList(id, filterBasedContingencyList));
+    }
+
+    @PutMapping(value = "/filters-contingency-lists/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Modify a filter based contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filter based contingency list have been modified successfully")})
+    public ResponseEntity<Void> modifyFilterBasedContingencyList(
+        @PathVariable UUID id,
+        @RequestBody FilterBasedContingencyList contingencyList,
+        @RequestHeader("userId") String userId) {
+        try {
+            service.modifyFilterBasedContingencyList(id, contingencyList, userId);
+            return ResponseEntity.ok().build();
+        } catch (EntityNotFoundException ignored) {
+            return ResponseEntity.notFound().build();
+        }
+    }
```

Comment:
PUT insert new lines in filter_metadata but not clean old lines, to verify annotation orphanRemoval option of OneToMany.
However, I see lines in relation table are removed